workshops/geoserver/intro/doc/source/install/geoserver.rst

```diff
@@ -0,0 +1,213 @@
+   
+   .. figure:: img/install_service.png
+      :width: 75%
+      
+      Install as service
+      
+#. When you are ready, click :guilabel:`Install` to start the installation.
+
+   .. figure:: img/install_ready.png
+      :width: 75%
+      
+      Ready to install
+
+#. After installation, click :guilabel:`Finish`.
+
+   .. figure:: img/install_finish.png
+      :width: 75%
+      
+      GeoServer has been installed
+
+#. Confirm GeoServer is working by visiting http://localhost:8080/geoserver in your browser.
+   
+   .. figure:: img/install_test.png
+      :width: 100%
+      
+      GeoServer Web Administration page
+
+Starting and stopping services
+------------------------------
+
+GeoServer has been installed as a windows service and is now running by default.  You can start and stop this services through the Start Menu.
+
+.. figure:: img/install_startstop.png
+   :width: 35%
+   
+   Start Menu entries for starting and stopping GeoServer
+
+You can also stop and start these services from the standard Windows Services dialog.
+
+Installing GeoServer Extensions
+-------------------------------
+
+The GeoServer installer provides the base GeoServer application. Optional functionality, known as extensions, are also available for download.
+
+This workshop requires the installation of several extensions:
+
+#. Navigate to the download page, or locate in the :file:`software` folder, the following extensions:
+
+   * :file:`geoserver-<version>-importer-plugin.zip`
+   * :file:`geoserver-<version>-wps-plugin.zip`
+   * :file:`geoserver-<version>-ysld-plugin.zip`
+
+   .. figure:: img/download_extensions.png
+   
+      Download extensions
+      
+#. Navigate to the GeoServer installation folder:
+   
+   * :file:`C:\\Program Files (x86)\\GeoServer`
+
+#. The GeoServer application is made up of :file:`jar` files located here:
+   
+   * :file:`C:\\Program Files (x86)\\GeoServer\\webapps\\geoserver\\WEB-INF\\lib`
+   
+#. Unzip each extension, placing additional :file:`jar` files into the :file:`lib` folder above.
+
+   * Choose :guilabel:`copy and replace` if there are any duplicated files.
+   
+#. Use :menuselection:`Start -> GeoServer -> Stop GeoServer` to stop the service.
+
+#. Use :menuselection:`Start -> GeoServer -> Start GeoServer` to start the service.
+
+#. Return to http://localhost:8080/geoserver to confirm the extensions are now available.
+   
+   * An additional service, ``WPS`` is now listed under :guilabel:`Service Capabilities`.
+   
+   .. figure:: img/install_extensions.png
+      :width: 100%
+      
+      GeoServer WPS extension
+
+.. note:: You may also hear of "community modules", community modules are experiments that require you to download the source code and compile. Some community modules are supported commercially (the mapbox style community module is included for download in Boundless Suite as a preview for our customers - but is not yet ready for wider release).

```

Comment:
Suggest: "You may also have heard of 'community modules'. These are experimental extensions that require you to download the source code and compile. Some community modules are supported commercially; for example, the MBstyle community module is included for download in Boundless Suite, but is not yet available for wider release to the community."